### PR TITLE
Add unstable release channel

### PR DESCRIPTION
**Pull request by doxavore**
Once we swap over to the new release channel, I'll follow up with removing `latest`.

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -6608,8 +6608,8 @@ function run() {
             core.setFailed("`access-token` field can't be empty.");
         }
         const releaseChannel = core.getInput('release-channel') || 'v1';
-        if (!['v1', 'latest'].includes(releaseChannel)) {
```
Comment (michaelglass) [inline dist/index.js:6611]: maybe we shouldn't document the name of the unstable channel if we want it to be internal only

File: .github/workflows/continuous_integration.yaml [hunk 2]
```diff
@@ -25,7 +25,7 @@ jobs:
       - uses: ./
         with:
           access-token: ${{ secrets.RWX_ACCESS_TOKEN }}
-      - run: abq --version | grep '^abq 0\.'
```
Comment (michaelglass) [inline .github/workflows/continuous_integration.yaml:28]: 🙏

**Review by michaelglass — APPROVED** [review]
looks good!

**Review by TAGraves — APPROVED** [review]

File: dist/index.js (hunk 1 above)
Comment (doxavore) [reply dist/index.js:6611]: Perhaps, but users would either need to be hosting their own ABQ instance or have us set them up for a hosted unstable version.

Maybe when I remove `latest` I can leave this check in but remove it from the help message. Then we'd be able to provide a meaningful message if another value is provided, without necessarily advertising that unstable is a thing.

Comment (michaelglass) [reply dist/index.js:6611]: minor suggestion either way, mostly food for thought

**Pull request merged**